package/lib/compileFunctions.js

```diff
@@ -62,6 +63,19 @@ module.exports = {
         });
       }
 
+      if (funcObject.vpcEgress) {
```

Comment:
I believe I might be missing something here - if we're only checking for setting on `funcObject`, does that mean that setting on provider level will always be ineffective?